api/features/versioning/managers.py

```diff
@@ -15,9 +16,25 @@
 class EnvironmentFeatureVersionManager(SoftDeleteManager):
     def get_latest_versions(self, environment: "Environment") -> RawQuerySet:
         """
-        Get the latest EnvironmentFeatureVersion objects
-        for a given environment.
+        Get the latest EnvironmentFeatureVersion objects for a given environment.
         """
         return self.raw(
             get_latest_versions_sql, params={"environment_id": environment.id}
         )
+
+    def get_latest_versions_as_queryset(
+        self, environment: "Environment"
+    ) -> QuerySet["EnvironmentFeatureVersion"]:
+        """
+        Get the latest EnvironmentFeatureVersion objects for a given environment
+        as a concrete QuerySet.
+
+        Note that it is often required to return the proper QuerySet to carry out
+        operations on the ORM object.
+        """
+        return self.filter(
+            uuid__in=[
+                efv.uuid
+                for efv in EnvironmentFeatureVersion.objects.get_latest_versions(self)
+            ]
+        )
```

Comment:
I'm not particularly happy with this, but the alternative was to update the sql to return all the fields and then (in the `Environment.clone` method) manually create the EnvironmentFeatureVersion, rather than using a clone method on the model class. To keep things consistent with other models, this seemed like the lesser of 2 evils.

Reply:
Why not write a second sql with all the fields?

> manually create the EnvironmentFeatureVersion, rather than using a clone method on the model class

I don't think so as long as the model instances were correctly mapped?

Reply:
I think that "Cloning environments" is not used frequently enough to make this a real issue.

Reply:
Fair point Novak

Reply:
>I think that "Cloning environments" is not used frequently enough to make this a real issue.

Well, I wish this were true. But the FE made the decision to remove the functionality to 'create' an environment, and now all environments are cloned from another. 

> Why not write a second sql with all the fields?

I'd still rather avoid this to avoid code duplication or hacky ways of concatenating SQL strings.